test/ccd2rgb.jl

```diff
@@ -44,4 +44,11 @@ end
     @test isapprox(red.(asinh_res), red.(asinh_ans), nans = true, rtol = 3e-5)
     @test isapprox(blue.(asinh_res), blue.(asinh_ans), nans = true, rtol = 3e-5)
     @test isapprox(green.(asinh_res), green.(asinh_ans), nans = true, rtol = 3e-5)
+
+    @testset "AstroImage using ccd2rgb" begin
```

Comment:
Probably test coverage can be improved for this new feature

Reply:
This part takes relatively large time compared to other tests. So, I wanted to keep non-redundant tests only. Also, in the next PR, I'm adding more fields and already written more tests in this very `testset`.

Reply:
I'm a bit worried by the fact this takes a lot of time. Does it mean that performance is that bad?

Reply:
Reproject is the bottleneck where it takes around 20 seconds to reproject an image of size `1024*1024`.

Reply:
This doesn't sound like a good news